src/config.rs

```diff
@@ -593,25 +593,31 @@ impl DefaultCfConfig {
             prop_keys_index_distance: self.prop_keys_index_distance,
         };
         cf_opts.add_table_properties_collector_factory("tikv.range-properties-collector", f);
-        match_template_api_version!(
-            API,
-            match api_version {
-                ApiVersion::API => {
-                    if API::IS_TTL_ENABLED {
-                        cf_opts.add_table_properties_collector_factory(
-                            "tikv.ttl-properties-collector",
-                            TtlPropertiesCollectorFactory::<API>::default(),
-                        );
-                        cf_opts
-                            .set_compaction_filter_factory(
-                                "ttl_compaction_filter_factory",
-                                TtlCompactionFilterFactory::<API>::default(),
-                            )
-                            .unwrap();
-                    }
-                }
+        match api_version {
```

Comment:
why do you remove the old proc_marco here? for developer reading code easily?

Reply:
> why do you remove the old proc_marco here? for developer reading code easily?

proc_marco is used to handle the same logic. But now the logic of V1ttl and V2 is different, so the proc_marco is not used